player/loadfile.c

```diff
@@ -1139,6 +1139,9 @@ static void play_current_file(struct MPContext *mpctx)
             startpos = start;
     }
     if (startpos != MP_NOPTS_VALUE) {
+        if (!opts->rebase_start_time) {
+            startpos += mpctx->demuxer->start_time;
```

Comment:
I meant adding `start_time` to the timestamps inside of `rel_time_to_abs` instead. `rel_time_to_abs` computes the start timestamp in the first place.

Not sure if that would make sense.